content/en/docs/chart_template_guide/named_templates.md

```diff
@@ -190,17 +190,18 @@ metadata:
     version:
 ```
 
-What happened to the name and version? They weren't in the scope for our defined
-template. When a named template (created with `define`) is rendered, it will
-receive the scope passed in by the `template` call. In our example, we included
+What happened to the name and version? The value of `.` was empty while our named template was rendered.
+When a named template (created with 
+`define`) is rendered without a second parameter, its `.` is empty. In our example, we included
 the template like this:
 
 ```yaml
 {{- template "mychart.labels" }}
 ```
 
-No scope was passed in, so within the template we cannot access anything in `.`.
-This is easy enough to fix, though. We simply pass a scope to the template:
+No pipeline value was provided to its second parameter, so within the template `.` is empty.
+`$.` will therefore also be empty.
```

Comment:
I think it should be `$` without the dot here.

Reply:
I see what you mean. I guess I got caught by the inconsistency that `.` is analogous to `$` but `.Foo` is analogous to `$.Foo` (extra dot appears from thin air) instead of `$Foo`. Something about `.` is magical I guess.

```
% echo "{{ . }}" | gomplate
map[]
% echo "{{ $. }}" | gomplate
16:41:50 ERR  error="template: -:1: unexpected <.> in operand"
```

Fixing...